src/util/api.test.js

```diff
@@ -0,0 +1,279 @@
+import * as API from './api';
```

Comment:
Może lepiej stworzyć folder o nazwie test, gdzie będziemy trzymać takie rzeczy na razie?

Reply:
Trzymałem się konwencji obowiązującej już w reszcie projektu (patrz np. src/components/Button.js i src/components/Button.test.js albo src/main.js i src/main.test.js). Też jestem osobiście za takim rozwiązaniem, ale to powinno być zrobione w ramach osobnego issue i PRa.

Reply:
Ja użyłem tej konwencji, bo w świecie JS częściej jest ona stosowana, ale jeżeli wolicie inną, to śmiało możecie ją zmienić w osobnym issue/PR tak jak napisał @tchojnacki.